2021/11/03 16:15:02 | python_logging.py(83 行) | INFO| 开始记录执行开始时间-------
2021/11/03 16:15:03 | python_logging.py(83 行) | INFO| 执行一共花费0.009127616882324219秒-------
2021/11/03 16:15:03 | python_logging.py(83 行) | INFO| 函数执行结果为3
2021/11/03 16:15:03 | python_logging.py(83 行) | INFO| 开始记录执行开始时间-------
2021/11/03 16:15:04 | python_logging.py(83 行) | INFO| 执行一共花费0.004042863845825195秒-------
2021/11/03 16:15:04 | python_logging.py(83 行) | INFO| 函数执行结果为12
2021/11/03 16:15:04 | python_logging.py(83 行) | INFO| 开始记录执行开始时间-------
2021/11/03 16:15:05 | python_logging.py(87 行) | ERROR| 执行函数过程中出错，错误原因为can't multiply sequence by non-int of type 'str'
